src/renderer/components/+config-priority-classes/legacy-store.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { asLegacyGlobalForExtensionApi } from "../../../extensions/as-legacy-globals-for-extension-api/as-legacy-global-object-for-extension-api";
+import priorityClassStoreInjectable from "./store.injectable";
+
+/**
+ * @deprecated use `di.inject(priorityClassStoreInjectable)` instead
+ */
+export const priorityClassStore = asLegacyGlobalForExtensionApi(priorityClassStoreInjectable);
```

Comment:
nit: unless you have some uses of this I wouldn't bother adding it.

Reply:
I didn't found single component without legacy-store.ts so if you have another idea how to not use `store` attribute in `React.Component` then maybe you can change it.

Reply:
https://github.com/lensapp/lens/blob/master/src/renderer/components/%2Bworkloads-pods/pods.tsx is a great example.

Reply:
Ok I refactored